docs/fundamentals/code-analysis/style-rules/naming-rules.md

```diff
@@ -19,15 +19,15 @@ helpviewer_keywords:
 ---
 # Naming rules
 
-Naming rules concern the naming of .NET programming language code elements, such as classes, properties, and methods. For example, you can specify that public members must be capitalized or that private fields must begin with `_`.
+Naming rules let you specify and enforce the naming of .NET programming language code elements, such as classes, properties, and methods. For example, you can specify that public members must be capitalized or that private fields must begin with `_`.
 
 A naming rule has three parts:
 
-* The group of symbols it applies to.
-* The naming style to associate with the rule.
+* The _symbol group_ it applies to. In our examples above, that's the "public members" or "private fields" portion of the rule.
```

Comment:
On this line and line 27, I'm not sure what the "examples above" refer to.

Reply:
The example on line 22. Would it help to maybe make those examples a separate list so they stand out?